floater/utils.py

```diff
@@ -218,3 +218,57 @@ def floats_to_castra(input_dir, output_fname, progress=False, **kwargs):
         if not c:
             c = Castra(output_fname, template=df)
         c.extend(df)
+
+def floats_to_netcdf(input_dir, output_fname, float_file_prefix):
+    """Convert MITgcm float data to NetCDF format.
+
+    Parameters
+    ----------
+    input_dir : path
+        Where to find the MITgcm output data
+    output_dir : path
+        Filename of the NetCDF data store
+    float_file_prefix: str
+        Prefix of MITgcm output files
+    """
+    import dask.dataframe as dd
+    import xarray as xr
+
+    float_digits = 10
+    float_columns = ['npart', 'time', 'x', 'y', 'z', 'u', 'v', 'vort']
+    float_dtypes = np.dtype([('npart', np.int32), ('time', np.float32), ('x', np.float32), ('y', np.float32), ('z', np.float32), ('u', np.float32), ('v', np.float32), ('vort', np.float32)])
+
+    reference_time = np.datetime64('1993-01-01', 'ns')
```

Comment:
I agree. None both reference_time and timestep should be keyword arguments to the function.

The time is then reference_date + delta_t * iteration_number. You might want to look at how this is done in xmitgcm:
https://github.com/xgcm/xmitgcm/blob/master/xmitgcm/mds_store.py#L769

If the user doesn't specify a timestep and a reference time, then the `time` dimension should just be an integer corresponding to the timestep number.